templater/models_test.go

```diff
@@ -0,0 +1,28 @@
+package templater_test
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	. "github.com/opencontrol/fedramp-templater/templater"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ControlTable", func() {
+	Describe("Fill", func() {
```

Comment:
I know you have a fixture at `simplified_table.xml` is there a way of getting that xml data and putting it in here as a string? I know docx is really ugly so the answer maybe no / not worth it.